### PR TITLE
Backups

### PR DESCRIPTION
**Pull request by ana0**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -114,11 +114,11 @@ services:
       - VIRTUAL_HOST=${HOST_API}
       - LETSENCRYPT_HOST=${HOST_API}
       - LETSENCRYPT_EMAIL=${SSL_EMAIL}
-    command: ./run.sh
```
Comment (adzialocha) [inline docker-compose.yml:117]: Upsi!

File: docker-compose.production.yml [hunk 2]
```diff
@@ -38,3 +38,24 @@ services:
       - VIRTUAL_HOST=${HOST_APP}
       - LETSENCRYPT_HOST=${HOST_APP}
       - LETSENCRYPT_EMAIL=${SSL_EMAIL}
+
+  backups: 
+    build:
```
Comment (adzialocha) [inline docker-compose.production.yml:43]: Is it possible to pull in the container by mentioning the image or is there a reason we need the Dockerfiles?

Comment (ana0) [reply docker-compose.production.yml:43]: I actually had to edit it significantly to get it working - I added support for multiple databases, updated the postgres version, updated to python3, etc

File: docker-compose.yml (hunk 1 above)
Comment (adzialocha) [reply docker-compose.yml:117]: This seems to be a merge mistake

**Pull request merged**